lib/elixir/lib/calendar/time.ex

```diff
@@ -115,7 +115,7 @@ defmodule Time do
           Calendar.hour(),
           Calendar.minute(),
           Calendar.second(),
-          Calendar.microsecond() | non_neg_integer,
```

Comment:
I'm not sure here, but the rest of the file uses `non_neg_integer()` spec notation, so I tweaked the spec here.